website/docs/run.md

```diff
@@ -64,6 +64,19 @@ and that Scala Native only supports Linux and macOS for now.
 scala-cli my-scala-native-app/ --native
 ```
 
+## ScalaScripts
```

Comment:
Why is this written as a single word?

Reply:
as I think now, it should be separate words